covid-19.md

```diff
@@ -12,37 +12,37 @@ research by sharing your own capacity.
 
 OSG is helping to organize extra resources to be made available to COVID-19 research!  These resources include:
 
+- **Capabilities for a variety of computational research tasks.** See [Computation on the OSG](https://opensciencegrid.org/about/computation/).
 - **Proactive consultation and facilitation** for quickly scaling up and automating workloads with up to [thousands of 
 concurrently-running jobs](https://gracc.opensciencegrid.org/dashboard/db/covid-19-research?orgId=1).
-- **Expertise** on using high throughput computing for maximizing research throughput, perhaps including 
-development effort.
-- **Access to computing capacity** through the *[OSG Connect](https://support.opensciencegrid.org/support/home)*
+- **Access to significant computing capacity** through the *[OSG Connect](https://support.opensciencegrid.org/support/home)*
   service, which provides a batch system submit environment based on HTCondor (allowing up to millions of jobs submitted via a
-  single submit file).  The OSG aggregates
-  about 100 clusters across the US and COVID-19 research could access tens of thousands of cores
-  through this mechanism.  The OSG also provides a CDN for distributing data to jobs.
-- **Capabilities for a variety of computational research tasks.** See [Computation on the OSG](https://opensciencegrid.org/about/computation/).
+  simple submit file).  The OSG aggregates
+  more than 100 clusters across the US, and COVID-19 projects have concurrently accessed tens of thousands of cores
+  through this mechanism.
```

Comment:
Did you mean to drop the sentence about the CDN?

Reply:
yes. Users won't know what that means, at all.

Reply:
How about we reword it instead? Data distribution through Stash Cache seems like something we should sell

Reply:
I don't think that level of detail is necessary here. The amount of data and fact that it can be automatedly distributed is covered above. What I've left here is specific to computational scale.